src/planner/binder/expression/bind_unpacked_star_expression.cpp

```diff
@@ -50,6 +50,10 @@ static void ReplaceInFunction(unique_ptr<ParsedExpression> &expr, expression_lis
 static void ReplaceInOperator(unique_ptr<ParsedExpression> &expr, expression_list_t &star_list) {
 	auto &operator_expr = expr->Cast<OperatorExpression>();
 
+	if (operator_expr.type == ExpressionType::OPERATOR_NOT) {
```

Comment:
We should probably also check `OPERATOR_IS_NULL/OPERATOR_IS_NOT_NULL` here

Reply:
Or maybe this should just be an opt-in instead? i.e. only make this work for `IN/NOT_IN`